src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,43 @@
 package mate.academy;
 
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
 public class EventManager {
+    private ExecutorService executor = Executors.newFixedThreadPool(5);
```

Comment:
Consider using a thread pool size that is configurable rather than hardcoding it.